<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initialization</class>
 <widget class="QWidget" name="initialization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>875</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>875</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>–êuthorization</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="db_name_Label">
       <property name="text">
        <string>DataBase name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_Label"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="db_ip_Label">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ip_Label"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="db_port_Label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="port_Label"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="db_login_Label">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="db_password_Label">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="password_Label"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="login_Label"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="locale">
      <locale language="Russian" country="Russia"/>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
